OracleDatabase/OracleConnectionManager/README.md

```diff
@@ -0,0 +1,56 @@
+Oracle Connection Manager on Docker
+
+==========================================
+
+Sample Docker build files to facilitate installation, configuration, and environment setup for DevOps users. For more information about Oracle Database please see the Oracle Connection Manager online Documentation. (http://docs.oracle.com/en/database/)
+
+1) How to build and run
+
+       This project offers sample Dockerfiles for:
+
+       Oracle Database 12c Release 2 Client (12.2.0.1.0) for Linux x86-64
+
+     	  To assist in building the images, you can use the buildDockerImage.sh script. The buildDockerImage.sh script is just a utility shell script that performs MD5 checks and is an easy way for beginners to get started. Expert users are welcome to directly call docker build with their prefered set of parameters.See below for instructions and usage.
+
+        IMPORTANT : Oracle Connection Manager container is useful when you want to bind single port to host and serve many container on different ports. Oracle Connection manager provide proxy conections. If you are running Oracle RAC Database on docker/container and network is not available for users, you can use Oracle Connection Manager image to proxy the connections.
+        For complete Oracle Connection Manager setup, please go though following steps and execute them as per your enviornment:
+
+2) You need to make sure that you have enough memory and cpu resources available for container.
+
+3) Create Oracle Connection Manager Image
+
+                IMPORTANT: You will have to provide the installation binaries of Oracle ADMIN Client Oracle Database 12c Release 2 Client (12.2.0.1.0) for Linux x86-64 and put them into the (DOCKER_CMAN_IMAGE)/dockerfiles/(VERSION) folder. You  only need to provide the binaries for the edition you are going to install. The binaries can be downloaded from the Oracle Technology Network. You also have to make sure to have internet connectivity for yum. Note that you must not uncompress the binaries. The script will handle that for you and fail if you uncompress them manually!
```

Comment:
The path seems wrong here. Also, what script are you referring to? You need to be able to build this image without using `buildDockerImage.sh`